src/main/scala/SBTS3Resolver.scala

```diff
@@ -129,12 +134,19 @@ object SbtS3Resolver extends AutoPlugin {
 
   // Default settings
   override def projectSettings: Seq[Setting[_]] = Seq(
-    awsProfile := "default",
+    awsProfile := None,
     s3credentials :=
-      new ProfileCredentialsProvider(awsProfile.value) |
-      new EnvironmentVariableCredentialsProvider() |
-      InstanceProfileCredentialsProvider.getInstance(),
-    s3region       := new com.amazonaws.regions.DefaultAwsRegionProviderChain(),
+      (awsProfile.value match {
+        case Some(profile) => new ProfileCredentialsProvider(profile)
+        case _ =>
+          new ProfileCredentialsProvider() |
```

Comment:
In the case of `None`, it should use `DefaultCredentialsProviderChain` from SDK. The same way as with the region. Sorry if I wasn't clear about it.

Reply:
I changed the code to use `DefaultAWSCredentialsProviderChain`.